Service Registry CLI guide (#308)
redhat-developer/app-services-guides: rhoas-cli-kafka/README.adoc

> ////

Author: This content is unchanged It is was just separated as separate chapter (kafka specific)

Reviewer: I don't see the install and login content here. I thought we were replicating the install and login in both getting strated guides for this release?

Author: We have 3 tutorials
CLI install
RHOAS 
RHOSAK

CLI install is basically independent and really lengthy. Putting it on beggining of every service guide is not good idea IMHO

Reviewer: Okay

START GENERATED ATTRIBUTES
WARNING: This content is generated by running npm --prefix .build run generate:attributes
////

:community:
:imagesdir: ./images
:product-version: 1
:product-long: Application Services
:product: App Services
:registry-product-long: OpenShift Service Registry
:registry: Service Registry
// Placeholder URL, when we get a HOST UI for the service we can put it here properly
:service-url: https://console.redhat.com/beta/application-services/streams/
:registry-url: https://console.redhat.com/beta/application-services/service-registry/
:property-file-name: app-services.properties

> :rhoas-version: 0.29.0

Reviewer 2: We may aswell update this now to the latest release.

Author: I never get why we doing it. This version is just informational right?

Reviewer 2: There's no real need, there was some discussion in https://github.com/redhat-developer/app-services-cli/issues/826#issuecomment-895524573 around removing it.

// Other upstream project names
:samples-git-repo: https://github.com/redhat-developer/app-services-guides

//URL components for cross refs
:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
:getting-started-url: getting-started/README.adoc
:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
:kafkacat-url: kafkacat/README.adoc
:quarkus-url: quarkus/README.adoc

> :rhoas-cli-url: rhoas-cli/README.adoc

Reviewer: So we need three new URLs here. One each for the cli getting starteds and one for the cli install?

Author: Who will be the best to ask how this works?

:rhoas-cli-ref-url: commands
:topic-config-url: topic-configuration/README.adoc
:consumer-config-url: consumer-configuration/README.adoc
:service-binding-url: service-discovery/README.adoc

////
END GENERATED ATTRIBUTES
////

[id="chap-getting-started-rhoas-cli"]

> = Getting started with the rhoas CLI and RHOSAK (Kafka)

Reviewer: ```suggestion
= Getting started with the OpenShift Streams for Apache Kafka CLI (rhoas )
```

:context: getting-started-rhosak

[IMPORTANT]
====
The `rhoas` command-line interface (CLI) is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
====

> 

Reviewer: With the `rhoas` command-line interface (CLI), you can manage your application services from a terminal. 

This guide describes how to get started quickly by using the `rhoas` CLI to create a simple Kafka instance, connecting to it, and creating a Kafka topic to produce and consume messages.

Reviewer: To do the three-way split, we need an install prereq. 

.Prerequisites
* You have downloaded and installed the latest version of the 'rhoas' CLI.
+
For instructions on installing the `rhoas` cli, see  information, see {base-url}{rhoas-cli-url}[_Installing the rhoas CLI _^]

Author: Do you mind adding this directly to make sure that we got this right
